trace_viewer/core/tracks/memory_dump_track.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 The Chromium Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="stylesheet" href="/core/tracks/memory_dump_track.css">
+
+<link rel="import" href="/base/extension_registry.html">
+<link rel="import" href="/core/trace_model/event.html">
+<link rel="import" href="/core/tracks/heading_track.html">
+<link rel="import" href="/core/color_scheme.html">
+<link rel="import" href="/base/sorted_array_utils.html">
+<link rel="import" href="/base/ui.html">
+
+<script>
+'use strict';
+
+tv.exportTo('tv.c.tracks', function() {
+  var SelectionState = tv.c.trace_model.SelectionState;
+  var EventPresenter = tv.c.EventPresenter;
+
+  /**
+   * A track that displays an array of memoryDump objects.
+   * @constructor
+   * @extends {HeadingTrack}
+   */
+  var MemoryDumpTrack = tv.b.ui.define(
+      'memory-dump-track', tv.c.tracks.HeadingTrack);
+
+  MemoryDumpTrack.prototype = {
+    __proto__: tv.c.tracks.HeadingTrack.prototype,
+
+    decorate: function(viewport) {
+      tv.c.tracks.HeadingTrack.prototype.decorate.call(this, viewport);
+      this.classList.add('memory-dump-track');
+      this.heading = 'Memory dumps';
+      this.memoryDumps = [];
+    },
+
+    get height() {
+      return window.getComputedStyle(this).height;
+    },
+
+    set height(height) {
+      this.style.height = height;
+    },
+
+    get dumpRadiusView() {
+      return 7 * (window.devicePixelRatio || 1);
```

Comment:
It's the radius of the point in pixels. Since it's essentially defined as a constant (because it's just a getter), I don't think it's absolutely necessary to add further indirection now, but if you prefer, I can give it a name (e.g. <tt>DUMP_RADIUS</tt>).